### ISSUE TITLE
Add support for different doc types for Year, Month and Date

### ISSUE THREAD

**Issue by LottePitcher** [Infocaster/Datefolders#3]
We would like to use this package on two sites that we are migrating from v7. One of these sites used [uDateFoldersy](https://our.umbraco.com/packages/backoffice-extensions/udatefoldersy/), the other used [SleepyMedia Date Folders](https://our.umbraco.com/packages/developer-tools/sleepymediadatefolders/), to give content editors the functionality that your package does.

However both of these packages allowed the site to use different document types for the Year, Month and Date, whereas it looks like this package expects the same document type to be used at each level.

Have you considered allowing alternate document types to be supported for each level, if needed? Then I believe your package could be a direct replacement for both uDateFoldersy and Sleepy Media Date Folders.

If you're up for this feature then I'd be up for trying a pull request! Thanks in advance.

**Comment by D-Inventor** [comment]
Hi @LottePitcher ! Thank you for reaching out! I think it's an awesome idea, but I should first pitch it with the rest of the team and see what they think. I'll get back to you on this before the end of this week.

**Comment by LottePitcher** [comment]
Thanks @D-Inventor !

**Comment by D-Inventor** [comment]
Hi again @LottePitcher ! Everybody loves your idea and we'd be very grateful if you're willing to send a pull request!

I'm looking forward to see what you come up with! 😊 If you run into any issues or if you have questions, you can find me here, on discord or on slack.

**Comment by LottePitcher** [comment]
Great, I'll give it a go!

**Comment by D-Inventor** [comment]
Hi @LottePitcher ! It's been a while since we've started this conversation!

Is this still something you wish to continue on? We're collecting some information currently on all our packages and looking for things we need to prioritize. Would love to hear your thoughts!

**Comment by LottePitcher** [comment]
Hey @D-Inventor , yes indeed it has been a while! 

The project I was working on this for got shelved unfortunately, so I didn't get any further with this. And no longer have the reason to. Sorry!

Should I close the draft PR and this issue?

**Comment by D-Inventor** [comment]
@LottePitcher yes, would appreciate if you could close the pr and issue. Still thank you for your effort, it's much appreciated.
